proposals/drafts/campaign_finance_filings.rst

```diff
@@ -0,0 +1,312 @@
+    Date of (nearest? next?) relevant election.
+
+filing_person
+    **optional**
+    Person responsible for the filing.
+
+Committee
+---------
+
+id
+    Open Civic Data-style id in the format ``ocd-cf-committee/{{uuid}}``
+
+name
+    Name of the Committee
+
+committee_type
+    Committee Type
+
+candidate orientations
+    **optional**
+    **repeated**
+    What posture the Committee takes towards specific Candidates. Committee type
+    can be imputed based on the number and nature of these orientations.
+
+officers
+    List of Persons who are the committee officers (maybe needs indication of
+    their ranks?)
+
+status
+    Current status of the Committee.
+
+purpose
+    **optional**
+    Purpose of the Committee if any is given.
+
+Committee Type
+--------------
+
+id
+    Open Civic Data-style id in the format ``ocd-cf-committeetype/{{uuid}}``
+
+name
+    Name of the Committee Type
+
+jurisdiction
+    Presumably a state, but: whatever jurisdiction the Committe Type is
+    meaningful within. This allows us to have "Candidate Committee"s in 2
+    different states that adhere to whatever different rules apply in those
+    places.
+
+Candidate Orientation
+---------------------
+
+A Committee may have no relation to any specific Candidate, but if they do have
+such a relationship, the options are complex. Hence this type.
+
+id
+    Open Civic Data-style id in the format ``ocd-cf-candidateorientation/{{uuid}}``
+
+candidate
+    Candidate
+
+orientation
+    Enumerated among "supports", "opposes", "primary vehicle for", "surplus
+    account for" and other relationship types.
+
+Person
+------
+
+This system assumes that each Person will be generated from a specific line item
+in a Filing. As such, we may know nothing about the Person but their name. Also,
+sometimes and as far as I can see inevitably, some Persons (many in fact) will
+be corporations or other distinctly non-human entities, Supremes Court
+notwithstanding.
+
+This type is an OCD Popolo Person.
+
+Regulator
+---------
+
+OCD Organization model.
+
+Amendment
+---------
+
+Need to figure out how this system should work. It probably shouldn't be a
+Section.
+
+Transaction Type
+----------------
+
+id
+    Open Civic Data-style id in the format ``ocd-cf-transactiontype/{{uuid}}``
+
+name
+    Type of transaction - contribution, expenditure, loan, in-kind, transfer,
+    other receipt, etc.
+
+Filing Type
+----------------
+
+id
+    Open Civic Data-style id in the format ``ocd-cf-filingtype/{{uuid}}``
+
+name
+    Name of filing type - "Last Minute Contributions", etc.
+
+code
+    Probably-more-cryptic code for the form associated with the Filing - "A1",
+    etc.
+
+jurisdiction
+    Jurisdiction for which the Filing Type is relevant.
+
+Transaction (Section)
+---------------------
+
+id
+    Open Civic Data-style id in the format ``ocd-cf-transaction/{{uuid}}``
+
+transaction_type
+    From enumerated list of possible Transaction Types.
+
```

Comment:
I think we may want to have a separate attribute that indicates in-kind versus cash.